account-kit/wallet-client/src/client/actions/getCallsStatus.ts

```diff
@@ -1,6 +1,6 @@
 import type { InnerWalletApiClient } from "../../types.ts";
-import type { Static } from "@sinclair/typebox";
-import { wallet_getCallsStatus } from "@alchemy/wallet-api-types/rpc";
+import type { Static } from "typebox";
+import type { wallet_getCallsStatus } from "@alchemy/wallet-api-types/rpc";
```

Comment:
Just out of curiosity: does this change allow us to omit the `typeof` keyword on line 7, or is the import still treated as a "type-only import of non-type item"?

Reply:
nope, still need that since it's still only exported as a value.